pkg/resources/standard_database_acceptance_test.go

```diff
@@ -0,0 +1,573 @@
+
+	secondaryAccountIdentifier := acc.SecondaryTestClient().Account.GetAccountIdentifier(t).FullyQualifiedName()
+
+	externalVolumeId, externalVolumeCleanup := acc.TestClient().ExternalVolume.Create(t)
+	t.Cleanup(externalVolumeCleanup)
+
+	catalogId, catalogCleanup := acc.TestClient().CatalogIntegration.Create(t)
+	t.Cleanup(catalogCleanup)
+
+	completeConfigVariables := func(
+		id sdk.AccountObjectIdentifier,
+		comment string,
+		dataRetention int,
+		maxDataExtension int,
+		replaceInvalidCharacters bool,
+		defaultDdlCollation string,
+		storageSerializationPolicy sdk.StorageSerializationPolicy,
+		logLevel sdk.LogLevel,
+		traceLevel sdk.TraceLevel,
+	) config.Variables {
+		return config.Variables{
+			"name":                            config.StringVariable(id.Name()),
+			"comment":                         config.StringVariable(comment),
+			"transient":                       config.BoolVariable(false),
+			"data_retention_time_in_days":     config.IntegerVariable(dataRetention),
+			"max_data_extension_time_in_days": config.IntegerVariable(maxDataExtension),
+			"external_volume":                 config.StringVariable(externalVolumeId.Name()),
+			"catalog":                         config.StringVariable(catalogId.Name()),
+			"replace_invalid_characters":      config.BoolVariable(replaceInvalidCharacters),
+			"default_ddl_collation":           config.StringVariable(defaultDdlCollation),
+			"storage_serialization_policy":    config.StringVariable(string(storageSerializationPolicy)),
+			"log_level":                       config.StringVariable(string(logLevel)),
+			"trace_level":                     config.StringVariable(string(traceLevel)),
+			"account_identifier":              config.StringVariable(secondaryAccountIdentifier),
+			"with_failover":                   config.BoolVariable(true),
+			"ignore_edition_check":            config.BoolVariable(true),
+		}
+	}
+
+	var (
+		dataRetentionTimeInDays    = new(string)
+		maxDataExtensionTimeInDays = new(string)
+		externalVolume             = new(string)
+		catalog                    = new(string)
+		replaceInvalidCharacters   = new(string)
+		defaultDdlCollation        = new(string)
+		storageSerializationPolicy = new(string)
+		logLevel                   = new(string)
+		traceLevel                 = new(string)
+	)
+
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
+		PreCheck:                 func() { acc.TestAccPreCheck(t) },
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.RequireAbove(tfversion.Version1_5_0),
+		},
+		CheckDestroy: acc.CheckDestroy(t, resources.StandardDatabase),
+		Steps: []resource.TestStep{
+			{
+				ConfigDirectory: acc.ConfigurationDirectory("TestAcc_StandardDatabase/basic"),
+				ConfigVariables: configVariables(id, comment),
+			},
+			{
+				PreConfig: func() {
+					params, err := acc.Client(t).Parameters.ShowParameters(context.Background(), &sdk.ShowParametersOptions{
```

Comment:
we could:
1. get rid of the previous step
2. replace parameter check for the db with show parameters for account (leaving the rest of the logic untouched)

Reply:
Removed first step and fetched acc parameters